README.md

```diff
@@ -129,6 +129,7 @@ Where `<your-image-prefix-here>` is something like `docker.io/brendanburns`.
        * `vs-kubernetes.namespace` - The namespace to use for all commands
        * `vs-kubernetes.kubectl-path` - File path to the kubectl binary. Note this is the binary file itself, not just the directory containing the file. On Windows, this must contain the `.exe` extension.
        * `vs-kubernetes.draft-path` - File path to the draft binary. Note this is the binary file itself, not just the directory containing the file. On Windows, this must contain the `.exe` extension.
+       * `vs-kubernetes.autoCleanupOnDebugTerminate` - The flag to control whether to auto cleanup the created Deployment and associated Pod by the command "Kubernetes: Debug (Launch)". The cleanup action occurs when it failed to start debug session or debug session terminated. If not specified, the extension will prompt for `"Clean Up"` or not.
```

Comment:
For last sentence suggest: "If not specified, the extension will prompt for whether to clean up or not. You might choose not to clean up in if you wanted to view pod logs, etc."